Headers/DebugServer2/Utils/CompilerSupport.h

```diff
@@ -12,26 +12,7 @@
 
 #include "DebugServer2/Base.h"
 
-// For GCC we define __has_attribute and __has_builtin to 1 because we know we
-// always have recent enough versions (support for C++11 for instance).
-
-#if !defined(__has_attribute)
-#if defined(COMPILER_GCC)
-#define __has_attribute(ATTR) 1
-#else
-#define __has_attribute(ATTR) 0
-#endif
-#endif
-
-#if !defined(__has_builtin)
-#if defined(COMPILER_GCC)
-#define __has_builtin(BUILTIN) 1
-#else
-#define __has_builtin(BUILTIN) 0
-#endif
-#endif
-
-#if __has_attribute(format)
+#if defined(COMPILER_GCC) || defined(COMPILER_CLANG)
```

Comment:
`#endif` is where?

Reply:
Wait, this becomes only for `__has_attribute(format)` now right?

Reply:
I don't understand the question :p

Reply:
I misunderstood what this was replacing. It looks fine to me now.